muffnn/mlp/tests/test_base.py

```diff
@@ -10,7 +10,7 @@
 import muffnn.mlp.base as base
 
 
-class TestEstimator(base.MLPBaseEstimator):
```

Comment:
i think "Test" is more appropriate than "Mock" in this context. it's not really like, say, a mock database connection, simulating complex behavior.  it's just a simple, useless implementation of a base class for testing